<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTicketDialog</class>
 <widget class="QDialog" name="AddTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Label_4">
      <property name="text">
       <string>Brand Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="brandLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>Model Item</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modelLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>Registration ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="registrationLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="Label_3">
      <property name="text">
       <string>Assigned Employee</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="employeeLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTicketDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTicketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
